Documentation/docs.md

```diff
@@ -4,7 +4,7 @@ etcd is a distributed key-value store designed to reliably and quickly preserve
 
 ## Getting started
 
-New etcd users and developers should get started by [downloading and building][download_build] etcd.
+New etcd users and developers should get started by [downloading and building][download_build] etcd. Here's a [quick demo][demo].
```

Comment:
Replace your addition with:
`Once you have etcd, follow this [quick demo][demo] to see the basics of creating and working with an etcd cluster.`